### PR TITLE
Extend `updateTokenInfo` test coverage

### LINKED ISSUE #149: Enhance updateTokenInfo test coverage
### Problem

 Add coverage for token keys defined as contractId or delegateContractId key types

### Solution

Implement above and verify for additional coverage options.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by georgi-l95**
**Description**:

- Adds extended test coverage for `updateTokenInfo` with tokens and different types of keys.
- Bump mirror node image version.
- Bump network node image version.

**Related issue(s)**:

Fixes #149 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by natanasow — APPROVED** [review]
LGTM

File: test/hts-precompile/token-managment/tokenManagmentContract.js [hunk 1]
```diff
@@ -254,4 +255,693 @@ describe("TokenManagmentContract tests", function () {
...
+
+            return updatedKey;
+        }
+
+        describe('Admin key set to ECDSA_secp256k', function() {
+            before(async function () {
+                tokenAddress = await utils.createFungibleTokenWithSECP256K1AdminKeyAssociateAndTransferToAddress(tokenCreateContract, tokenCreateContract.address, utils.getSignerCompressedPublicKey());
+                tokenInfoBefore = await getTokenInfo(tokenQueryContract, tokenAddress);
+        
+                await utils.associateToken(tokenCreateContract, tokenAddress, 'TokenCreateContract');
```
Comment (lukelee-sl) [inline test/hts-precompile/token-managment/tokenManagmentContract.js:303]: nit: is this associate and grantKyc necessary as it is being done in the contract?

File: test/hts-precompile/token-managment/tokenManagmentContract.js [hunk 2]
```diff
@@ -254,4 +255,693 @@ describe("TokenManagmentContract tests", function () {
...
+                            ...tokenInfoBefore, tokenKeys: [[ utils.KeyType.PAUSE, updatedKey]]
+                        };
+                        token.treasury = signers[0].address;
+    
+                        await updateTokenInfo(tokenManagmentContract, tokenAddress, token);
+                    }
+                    
+                    //Pause and unpause token
+                    {
+                        const pauseTokenTx = await tokenManagmentContract.connect(signers[1]).pauseTokenPublic(tokenAddress);
```
Comment (lukelee-sl) [inline test/hts-precompile/token-managment/tokenManagmentContract.js:324]: Is there a way we can know the the token has been paused/unpaused beyond the SUCCESS return value?

File: test/hts-precompile/token-managment/tokenManagmentContract.js (hunk 1 above)
Comment (georgi-l95) [reply test/hts-precompile/token-managment/tokenManagmentContract.js:303]: This is a util function, which grants KYC and associate signers[0] and signer[1] to the token, which is needed for the tests.

File: test/hts-precompile/token-managment/tokenManagmentContract.js (hunk 2 above)
Comment (georgi-l95) [reply test/hts-precompile/token-managment/tokenManagmentContract.js:324]: Yep, I will add those and request review again.

Comment (georgi-l95) [reply test/hts-precompile/token-managment/tokenManagmentContract.js:324]: Done.

**Review by lukelee-sl — APPROVED** [review]
LGTM

**Pull request merged**